Moda.Web/src/moda.web.reactclient/src/app/components/common/planning/active-program-increments.tsx

```diff
@@ -1,63 +1,51 @@
 import { useEffect, useState } from 'react'
 import PageTitle from '../page-title'
-import { getProgramIncrementsClient } from '@/src/services/clients'
 import { Col, Divider, Row, Space } from 'antd'
 import ProgramIncrementCard from './program-increment-card'
 import dayjs from 'dayjs'
+import { useGetProgramIncrements } from '@/src/services/queries/planning-queries'
 
 const isWithinTwoWeeks = (date: Date) => {
   const twoWeeksFromNow = dayjs().add(14, 'day')
   return dayjs(date) <= twoWeeksFromNow
 }
 
 const ActiveProgramIncrements = () => {
+  const { data: piData } = useGetProgramIncrements()
   const [activeProgramIncrements, setActiveProgramIncrements] = useState([])
 
   useEffect(() => {
-    const loadActiveProgramIncrements = async () => {
-      const programIncrementClient = await getProgramIncrementsClient()
-      const programIncrementDtos = await programIncrementClient.getList()
-      const activeProgramIncrements = programIncrementDtos.filter(
+    const activeProgramIncrements = piData
+      ?.filter(
         (pi) =>
           pi.state === 'Active' ||
-          (pi.state === 'Future' && isWithinTwoWeeks(pi.start))
+          (pi.state === 'Future' && isWithinTwoWeeks(pi.start)),
       )
-      setActiveProgramIncrements(
-        activeProgramIncrements?.sort((a, b) => {
-          return new Date(a.start).getTime() - new Date(b.start).getTime()
-        })
-      )
-    }
-
-    loadActiveProgramIncrements()
-  }, [])
-
-  const hasActiveProgramIncrements = activeProgramIncrements.length > 0
-
-  function RenderContent() {
-    if (hasActiveProgramIncrements) {
-      return (
-        <>
-          <Row>
-            <PageTitle title="Program Increments" />
-          </Row>
-          <Row>
-            <Space>
-              {activeProgramIncrements.map((pi) => (
-                <Col key={pi.localId}>
-                  <ProgramIncrementCard programIncrement={pi} />
-                </Col>
-              ))}
-            </Space>
-          </Row>
-          <Divider />
-        </>
-      )
-    }
-    return <></>
-  }
-
-  return <RenderContent />
+      ?.sort((a, b) => {
+        return dayjs(a.start).unix() - dayjs(b.start).unix()
```

Comment:
Not a big deal, but you can also sort by doing new Date(a.start) - new Date(b.start).

Reply:
that throws a typescript error similar to using dayjs() without the .unix().